README.md

```diff
@@ -18,115 +26,317 @@ feedback, please join us there! We look forward to your contribution.
+
+```
+# salt-run state.orch ceph.stage.3
+```
+or
+```
+# salt-run state.orch ceph.stage.deploy
+```
+
+The command may take several minutes. If it fails, you have to fix the issue and run the previous stages again. After the command succeeds, run the following to check the status:
+
+```
+# ceph -s
+```
+
+If you only want to deploy a Ceph cluster without any additional services,
+congratulations - you're done. Otherwise you have to continue with Stage 4.
+
+#### Stage 4
+The last step of the Ceph cluster deployment is the services stage. Here you
+instantiate any of the currently supported services: iSCSI Gateway, CephFS,
+RADOS Gateway, openATTIC, and NFS Ganesha. In this stage, the necessary pools,
+authorizing keyrings and starting services are created.
+
+To start the stage, run the following:
+
+```
+# salt-run state.orch ceph.stage.4
+```
+or
+
+```
+salt-run state.orch ceph.stage.services
+```
+
+Depending on the setup, the command may run several minutes. If you specified
+the openATTIC role it will be installed on the master node by default. If you
+need to install openATTIC on a different node, please take a look at the
+upstream [openATTIC
+documentation](http://docs.openattic.org/en/latest/install_guides/). If you do
+not need to install openATTIC at all remove the role from your `policy.cfg`.
 
 ## Test intial deployment and generating load
 Once a cluster is deployed one might want to verify functionality or run
 benchmarks to verify the cluster works as expected.
-- In order to gain some confidence in your cluster after the inital deployment
-  (stage 3) run `salt-run state.orch ceph.benchmarks.baseline`. This runs an osd
-  benchmark on each OSD and aggregates the results. It reports your average OSD
-  performance and points out OSDs that deviate from the average. *Please note
-  that for now the baseline benchmark assumes all uniform OSDs.*
-- To load test CephFS run `salt-run state.orch ceph.benchmarks.cephfs`.
-  This requires a running MDS (deploy in stage 4) and at least on minion with
-  the mds-client role. The cephfs_benchmark stage will then mount the CephFS
-  instance on the mds-client and run a bunch of fio tests. See the [benchmark
-  readme](srv/pillar/ceph/benchmark/README.md) for futher details.
-- *more to come*
+
+In order to gain some confidence in your cluster after the inital deployment
+(stage 3) run: 
+
+```
+# salt-run state.orch ceph.benchmarks.baseline
+``` 
+
+This runs an osd benchmark on each OSD and aggregates the results. It reports
+your average OSD performance and points out OSDs that deviate from the average. 
+
+*Please note that for now the baseline benchmark assumes all uniform OSDs.*
+
+To load test CephFS run:
+
+```
+# salt-run state.orch ceph.benchmarks.cephfs
+```
+
+This requires a running MDS (deploy in stage 4) and at least on minion with the
+`mds-client` role. The `cephfs_benchmark` stage will then mount the CephFS
```

Comment:
The client role name changed recently to ```client-cephfs```. Could you please add this change here?